tools/release/bump-version-and-create-release-note.sh

```diff
@@ -35,6 +35,10 @@ docker run --rm -it -v "$(pwd)":/usr/local/src/your-app ferrarimarco/github-chan
 # bump the version
 echo "${RELEASE_VERSION}" >VERSION
 
+#Update aws-otel-collector msi file. 
+# Note: Only supports [0-9]*.[0-9]*.[0-9]* version pattern. If we decide to release with a full sem ver vX.XX.XX-prerelease then the .wxs file will need to be manually updated
+sed -i '' "s/^Version\=\"[0-9]*.[0-9]*.[0-9]*\"/Version=\"${RELEASE_VERSION:1}\"/" tools/packaging/windows/aws-otel-collector.wxs
```

Comment:
```suggestion
sed -i '' "s/^Version\=\"[0-9]+.[0-9]+.[0-9]+\"/Version=\"${RELEASE_VERSION:1}\"/" tools/packaging/windows/aws-otel-collector.wxs
```

`...` is not a valid version...

Reply:
Good catch! 🤦  That's on me!